blog/settings.py

```diff
@@ -0,0 +1,122 @@
+"""
+Django settings for blog project.
```

Comment:
I would leave just this line. Information below isn't particularly helpful or useful.